components/ui/command-bar/autocomplete-input/autocomplete-input.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect, createRef, InputHTMLAttributes } from 'react';
+import classNames from 'classnames';
+import styles from './autocomplete-input.module.scss';
+
+type AutoCompleteInputProps = InputHTMLAttributes<HTMLInputElement> & {
+  focus?: Truthy;
+};
+
+type Truthy = boolean | number | string | null | undefined;
```

Comment:
It's really confusing to have type called Truthy that can get null and undefined:)

Reply:
truthy as in https://developer.mozilla.org/en-US/docs/Glossary/Truthy